desktop/components/main_layout/templates/index.jade

```diff
@@ -9,6 +9,7 @@ html(
   data-useragent=userAgent
   data-user-type=sd.CURRENT_USER && sd.CURRENT_USER.type
   data-lab-features=sd.CURRENT_USER && sd.CURRENT_USER.lab_features && sd.CURRENT_USER.lab_features.join(',')
+  data-collect-keyword-search=sd.COLLECT_KEYWORD_SEARCH
```

Comment:
This makes it easier to conditionally style a lil' CSS thing (using the selector `html[data-collect-keyword-search='experiment']` above)